ipatests/prci_definitions/nightly_ipa-4-6.yaml

```diff
@@ -1134,3 +1134,15 @@ jobs:
         template: *ci-master-f27
         timeout: 7200
         topology: *adroot_adchild_adtree_master_1client
+
+  fedora-27/test_cert:
+    requires: [fedora-27/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-27/build_url}'
+        test_suite: test_integration/test_cert.py
+        template: *ci-master-27
```

Comment:
Missing letter `f`, it should be: `*ci-master-f27`

Reply:
Fixed. I made the same mistake in gating.yaml